tools/cpplint.py

```diff
@@ -541,6 +545,11 @@ def SetFilters(self, filters):
         raise ValueError('Every filter in --filters must start with + or -'
                          ' (%s does not)' % filt)
 
+  def setOutputFile(self, filename):
+    """attempts to create a file which we write output to."""
+    fh = logging.FileHandler(filename, mode='wb')
+    logger.addHandler(fh)
```

Comment:
Shouldn't we set the log level here? By default, it will be printing only the warnings, right?